src/main/java/com/microsoft/azure/vmagent/AzureVMCloud.java

```diff
@@ -798,6 +806,19 @@ public static void addAliases() {
             Jenkins.XSTREAM2.addCompatibilityAlias("com.microsoft.azure.Messages", Messages.class);
         }
 
+        @Initializer(before = PLUGINS_STARTED)
+        public static void addLogRecorder(Jenkins h) throws IOException {
+            LogRecorderManager manager = h.getLog();
+            Map<String, LogRecorder> logRecorders = manager.logRecorders;
+            if (!logRecorders.containsKey(LOG_RECORDER_NAME)) {
+                LogRecorder recorder = new LogRecorder(LOG_RECORDER_NAME);
+                String packageName = AzureVMAgent.class.getPackage().getName();
+                recorder.targets.add(new LogRecorder.Target(packageName, Level.WARNING));
```

Comment:
This is the default level, user can change the log level later in logger config, right?